RecoTracker/TkSeedGenerator/plugins/SeedGeneratorFromL1TTracksEDProducer.cc

```diff
@@ -21,6 +21,7 @@
 
```

Comment:
this is the only file in Reco packages.
It sounds like DigiGeometry is preferred after all for the HLT setup.
If so, it may be better to update this file separate from the larger set of updates in  HLTrigger/Configuration
(one less signature to collect)

Reply:
This is the comment from @perrotta above, I still have to tackle that!

Reply:
I cherry-picked #30990, but it didn't solve the conflict. Anyway the offending file is removed, when I clean up the PR it will need less signatures to collect!

Reply:
a rebase with the commit in RecoTracker removed would be more appropriate